releasenotes/notes/support-empty-delete-for-pauli-16c5c5fae890c16c.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Passing empty list to delete method of Pauli, PauliList, and PauliTable returns a copy of the original object instead of throwing an error.
```

Comment:
```suggestion
    Passing an empty list to the methods :meth:`.Pauli.delete`, :meth:`.PauliList.delete`,
    and :meth:`.PauliTable.delete` now returns a copy of the original object instead of throwing an error.
```